doc/user/content/self-hosted/_index.md

```diff
@@ -0,0 +1,262 @@
+| **Instance types**                           | **r6g**, **r7g** families <br> **Note:** LVM setup may work on other instance types with local storage (like i3.xlarge, i4i.xlarge, r5d.xlarge), but we have not extensively tested these configurations. |
+| **AMI**                                      | AWS Bottlerocket |
+| **Instance store volumes**                   | Required |
+
+To setup:
+
+1. Use Bottlerocket bootstrap container for LVM configuration.
+1. Configure volume group name as `instance-store-vg`
+
+
+
+#### Storage Configuration
+
+Once LVM is configured, set up the storage class:
+
+{{< tabs >}}
+{{< tab "OpenEBS" >}}
+
+If using OpenEBS, set up the storage class as follows:
+```yaml
+storage:
+  storageClass:
+    create: true
+    name: "openebs-lvm-instance-store-ext4"
+    provisioner: "local.csi.openebs.io"
+    parameters:
+      storage: "lvm"
+      fsType: "ext4"
+      volgroup: "instance-store-vg"
+```
+{{< /tab >}}
+{{< tab "Other Storage" >}}
+While OpenEBS is our recommended solution, you can use any storage provisioner that meets your performance requirements by overriding the provisioner and parameters values.
+
+For example, to use a different storage provider:
+
+```yaml
+storage:
+  storageClass:
+    create: true
+    name: "your-storage-class"
+    provisioner: "your.storage.provisioner"
+    parameters:
+      # Parameters specific to your chosen storage provisioner
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+## Installing the Chart
+
+1. If installing for the first time, create a namespace. The default
+   configuration uses the `materialize` namespace.
+
+   ```bash
+   kubectl create namespace materialize
+   ```
+
+1. Install the Materialize operator with the release name
+   `my-materialize-operator`
+
+      - To use the [default configuration](/self-hosted/configuration/):
+
+        ```shell
+        helm install my-materialize-operator materialize/misc/helm-charts/operator
+        ```
+
+      * To include custom configuration, you can:
+
+        - *Recommended:* Create or edita a YAML file (such as the sample
+          `values.yaml` file located in the
+          `materialize/misc/helm-charts/operator/` directory) that specifies the
+          values for the parameters and then install the chart with the `-f`
+          flag:
+
+          ```shell
+          helm install materialize/misc/helm-charts/operator/values.yaml  materialize/misc/helm-charts/operator
+          ```
+
+        - Specify each parameter using the `--set key=value[,key=value]`
+            argument to `helm install`. For example:
+
+            ```shell
+            helm install my-materialize-operator \
+              --set operator.image.tag=v1.0.0 \
+              materialize/materialize-operator
+            ```
+
+1. Verify the installation:
+
+    ```shell
+    kubectl get all -n materialize
+    ```
+
+## Deploying Materialize
+
+### Set up the metadata database
+
```

Comment:
Let me know if you want me to stub this with our testing yaml  file.

Reply:
Will add generic content here tomorrow.  But wanted to push up quickly the information architecture changes along with the self-hosted -> self-managed nomenclature changes.